styles.css

```diff
@@ -99,3 +102,136 @@ p {
   gap: 20px;
   margin-top: 10px;
 }
+
+.social-nl a {
+  margin-right: 10px;
+  color: #505f79;
+  font-size: 25px;
+}
+
+.social a:hover {
+  color: #505f79;
+  opacity: 0.5;
+}
+
+/* Work */
+.work .card {
+  flex-direction: row;
+  gap: 40px;
+}
+
+.skills .skill li span h2 {
+  font-weight: 500;
+  font-size: 20px;
+  color: #000;
+}
+
+.work .card .cano {
+  margin-top: 10px;
+}
+
+.skill-lang li {
+  display: flex;
+  flex-direction: column;
+  justify-content: start;
+  background: #f7f7f9;
+  border-radius: 8px;
+
+  /* width: 100%; */
+  width: 122px;
+  margin: 5px 0;
+  margin-right: 10px;
+  padding: 10px;
+}
+
+.work .card .cano li {
+  font-size: 18px;
+}
+
+.contact p {
+  font-size: 20px;
+  width: 42%;
+  margin: 10px auto;
+}
+
+.work .card p {
+  font-size: 16px;
+  margin-top: 10px;
+}
+
+.work .card .lang {
+  margin-top: 10px;
+}
+
+.contact button {
+  display: inline;
+  margin: 10px auto;
+}
+
+.work .card button {
+  margin-top: 10px;
+}
+
+.work .card-left-mobile {
+  display: none;
+}
```

Comment:
- [ ] From the project requirements, you should stick to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0), But as you can see there are some differences between work section of the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0) and your design so make sure that you adjust the styling (text-font, layout, spacing, font-size, element alignment, button style, margins) of all your project cards to look exactly like the Figma design. Ensure that you have a margin between the headline and work section as it is in the Figma design :pray: 
  
  | Your Design |Figma Design |
  | :---:   | :---: | 
  | ![image](https://github.com/Srilearns/Portfolio-Website-Mobile-First/assets/92303891/bc1b26be-bb8e-437f-85ad-41dc1d43c7ef) | ![image](https://github.com/Srilearns/Portfolio-Website-Mobile-First/assets/92303891/571424a0-15a5-4019-9132-5e21703345e5)|

- [ ] - [ ] Please ensure that all buttons (e.g., "Get my resume", "See Projects") follow the button styling: Active, Hover, and Pressed (see design :point_down: )

![image](https://github.com/Srilearns/Portfolio-Website-Mobile-First/assets/92303891/f19591f7-608c-4669-88ab-2e4f8a077b4e)